### ISSUE TITLE
lets get rid of our junk!

### ISSUE THREAD

**Issue by dominictarr** [ssb-junkyard/junkyard#1]
I was looking for more modules to add to the village tracker process...

and I saw we had a lot of junk. There is lots of old stuff that isn't documented, doesn't do anything, maybe is interesting from a historical perspective, but isn't important to someone trying to understand how ssb works _currently_, and actually is a obstical to that, because you have to figure out if it's still relevant.

You can archive a repo, but it's still shown in the repos list, so you still have to scroll past it. better to move it here.

**Comment by dominictarr** [comment]
@christianbundy @mixmix @ahdinosaur @marylychee @clevinson I invited you all as colaborators, I made mix owner. we need everyone to be owners so that they can add more people. if you add transfer something here, remember to _turn off issues_ (which is inside settings, somewhere)

**Comment by mixmix** [comment]
Can you seed this with some examples of modules that are being junkyarded?

It might be fun to get together on a call and point to old junk. I'm excited to clearly deprecate / sunset some modules huh

**Comment by dominictarr** [comment]
@mixmix I have done added several modules already. If I added something you disagreed about, feel free to move it back. I think the time of a call could be better spent just doing it, but if you want to be on a call while you do it go ahead.

**Comment by dominictarr** [comment]
Adding this as issue in village tracker. if you got this some one will need to add you as owner of this org and ssbc, so you can transfer modules

**Comment by dominictarr** [comment]
the task is look for things that are not used anymore, not useful, and/or no longer maintained, and move them here.

**Comment by christianbundy** [comment]
Another option would be to archive the repo. This would keep the URL but make the repo read-only, plus it shows a warning to folks trying to interact with it. More info [here](https://help.github.com/en/articles/archiving-a-github-repository)

**Comment by dominictarr** [comment]
@christianbundy yes I tried that but it still appears in the repository list so it doesn't de-clutter

**Comment by dominictarr** [comment]
we should still archive the modules... I just wish there was a way to archive in bulk
